tests/integration/test_httpx.py

```diff
@@ -40,6 +42,9 @@ def client(self):
 
     def __call__(self, *args, **kwargs):
         return self.client.request(*args, timeout=60, **kwargs)
+    
+    def stream(self, *args, **kwargs):
+        return self.client.stream(*args, timeout=60, **kwargs)
```

Comment:
When trying to reproduce the issue locally in a way similar to your test `test_simple_stream_fetching`, this method did not seem to be needed or called. Could you help me understand its purpose?

Reply:
This was an attempt to make a test using the existing DoRequest classes. It was easier just to make a new client and use that as the object tested. It can be deleted.